doc/semantics-of-the-publishing-api.md

```diff
@@ -0,0 +1,406 @@
+
+---
+
+### routes
+
+Example: *[{ path: “/vat-rates”, type: "exact" }]*
+
+Required: Conditionally
+
+The `routes` are used to configure the GOV.UK router for the content item. The
+`routes` appear as a JSON array and each element in this array is a JSON object
+that contains the properties *path* and *type*. No other properties are
+supported.  The *type* must be set to *exact* to denote that the route maps to
+an exact path on the GOV.UK website.
+
+The `routes` are generally required, except for the case when the content item
+has a `format` of *redirect*. In this case, the `routes` must not be present as
+it doesn’t make sense to have routes for a redirect. When the `format` is
+anything except *redirect*, the routes must include the `base_path` of the
+content item, at the very least.
+
+If additional `routes` are specified other than the one for the `base_path`, all
+of these `routes` must reside under the `base_path`. Here is an example:
+
+```
+[
+  { path: “/vat-rates”, type: "exact" },
+  { path: “/vat-rates/tax-thresholds”, type: "exact" },
+  { path: “/vat-rates/more-resources”, type: "exact" }
+]
+```
+
+Note: Collectively, routes and redirects must have unique paths. The Publishing
+API will not accept content items where the routes and redirects conflict with
+each other.
+
+---
+
+### redirects
+
+Example: *[{ path: “/vat-rates/tax-thresholds”, type: "exact", destination: “/vat-rates/bands” }]*
+
+Required: Conditionally
+
+The `redirects` are used to configure the GOV.UK router to specify redirects
+related to the content item. The `redirects` appear as a JSON array and each
+element in this array is a JSON object that contains the properties *path*,
+*type* and *destination*. No other properties are supported. The *type* must be
+set to one of either *exact* or *prefix*.
+
+An *exact type* denotes that the *path* should be checked for an exact match
+against the user’s request URL when determining whether a redirect should occur.
+A *prefix type* denotes that any subpath under the specified path should be
+redirected to the destination. You can think of these as “exact” and “fuzzy”
+matches, respectively.
+
+The `redirects` are generally optional, except for the case when the content
+item has a `format` of *redirect*. In this case, the redirects must be present
+and they must include the `base_path` of content_item in the *path* property, at
+the very least.
+
+Redirects are subject to the same requirement as routes in that their paths must
+reside under the `base_path` of the content item (see [**routes**](#routes)).
+
+Note: Collectively, routes and redirects must have unique paths. The Publishing
+API will not accept content items where the routes and redirects conflict with
+each other.
+
+---
+
+### update_type
+
+Example: *major*
+
+Required: Conditionally
+
+The `update_type` is an indicator of how much the content item has changed since
+the last time it was published. This field is required when publishing the
```

Comment:
It's more about the importance of the change than the amount.  It could be two words but if it has a significant impact on the readers it would be a major change.  The litmus test is whether the readers would want to be notified about the change or not.